.github/workflows/e2e-release.yml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "End to End - Release"
+on:
+  schedule:
+    # run every day at 8pm
+    - cron:  '* 20 * * *'
```

Comment:
8 PM of which time zone?

Reply:
Good point, it looks like GitHub Actions uses UTC. I don't think the timing matters too much though, just want to make sure it runs frequently